connect/client_test.go

```diff
@@ -6,13 +6,52 @@ import (
 	"testing"
 	"time"
 
+	"github.com/stretchr/testify/assert"
 	"github.com/stretchr/testify/require"
 
 	cluster_api "github.com/pomerium/zero-sdk/cluster"
 	"github.com/pomerium/zero-sdk/connect"
 	"github.com/pomerium/zero-sdk/token"
 )
 
+func TestConfig(t *testing.T) {
+	t.Parallel()
+
+	for _, tc := range []struct {
+		endpoint      string
+		connectionURI string
+		requireTLS    bool
+		expectError   bool
```

Comment:
(optional)

Would it make sense for this to be a string (the error message itself)? (Totally up to you.)